examples/blog-api/src/api/util.rs

```diff
@@ -2,14 +2,13 @@ use diesel::result::{DatabaseErrorKind, Error as DieselError};
 use http::StatusCode;
 use via::error::Sanitizer;
 
-/// Sanitizes details about database errors that
-pub fn with_error_sanitizer(sanitizer: &mut Sanitizer) {
+pub fn error_sanitizer(sanitizer: &mut Sanitizer) {
     // Print the original message to stderr. In production you probably want
     // to use env_logger, tracing, or something similar.
     eprintln!("error: {}", sanitizer);
 
     // Configure the sanitizer to generate a JSON response.
-    sanitizer.respond_with_json();
+    sanitizer.use_json();
```

Comment:
`Rescue::with` reads much better than `Rescue::new`. In the context of `respond_with_json`, it would become redundant. `set_*` and `use_*` make more sense in this particular context.